mango/tests/processing_module/test_file_functions.py

```diff
@@ -103,6 +113,19 @@ def test_excel_file(self):
             self.assertIsInstance(data["Sheet1"], list)
             self.assertEqual(data["Sheet1"], [{"a": 1, "b": 2}, {"a": 3, "b": 4}])
 
+        # Test pandas ImportError
+        with mock.patch.dict("sys.modules", {"pandas": None}):
+            with warnings.catch_warnings(record=True) as warning_list:
+                # Run the function that may raise a warning
+                result = load_excel(file)
+
+                # Assert that the warning is raised
+                self.assertEqual(len(warning_list), 1)
+                self.assertEqual(
+                    str(warning_list[0].message),
+                    "pandas is not installed so load_excel_open will be used. Data can only be returned as list of dicts.",
+                )
```

Comment:
We should check that results has the correct format

Reply:
Added call to self.test_excel_light() to solve this